OpenImis.ModulesV3/InsureeModule/Models/EnrollFamilyModels/Policy.cs

```diff
@@ -18,6 +18,7 @@ public class Policy
         public int OfficerId { get; set; }
         public string PolicyStage { get; set; }
         public string isOffline { get; set; }
+        public int ControlNumberId { get; set; }
```

Comment:
Do you think we need to set this attribute as optional (as int?) to cover the context not using ePayment?

Reply:
Because we are not using ControlNumberId for inserts, int is fine. it will come with value 0 and we can use the condition if the value is greater than 0 rather than first checking if the value is null

Reply:
I just realized that you used the ID and not the actual CN. Please change this to string and use the actual CN instead of numerical value. Change the name to ControlNumber also and make the necessary adaptation in the code. 
Kamil also makes same changes. 
Thanks.

Reply:
This was discussed with Kamil beforehand. Because he has both the IDs and Control numbers in his local database, we decided to pass the ControlNumberId for the better performance. ControlNumberId is a primary key and will be better to use in where criteria than the string (Control Number) comparision

Reply:
This will work when you have available CN in mobile app. But if you don't have and need to request through USSD and set it manually, then you don't have the ID. We need to handle all cases.